language/en-GB/com_media.ini

```
@@ -92,6 +92,7 @@ COM_MEDIA_MEDIA_SIZE="Size"
 COM_MEDIA_MEDIA_TYPE="Type"
 COM_MEDIA_NAME="Image Name"
 COM_MEDIA_NO_IMAGES_FOUND="No Images Found"
+COM_MEDIA_NO_MEDIA_FOUND="No Media Added"
```

```suggestion
COM_MEDIA_NO_MEDIA_FOUND="No Media Found"
```

For consistency please use the same verb as on the previous line